recipes-core/packagegroups/packagegroup-arago-tisdk-addons.bbappend

```diff
@@ -14,11 +14,6 @@ DEMOS:append:am64xx = " \
     opcua-server \
 "
 
-EXTRA_PACKAGES:append = " \
-    watchdog \
-    kmscube \
-"
-EXTRA_PACKAGES:remove:am64xx = "kmscube"
```

Comment:
@cshilwant Are you sure we want to delete this Removal too?
looks like [here] it is added unconditionally

[here]: https://git.ti.com/cgit/arago-project/meta-arago/commit/?h=scarthgap-next&id=602b8eec40aedacc07e84b31333f9bc273addd6d

Reply:
@aniket-l `packagegroup-arago-tisdk-graphics` gets added to `tisdk-default-image` only when [DISTRO_FEATURES contains opengl](https://git.ti.com/cgit/arago-project/meta-arago/tree/meta-arago-distro/recipes-core/images/tisdk-default-image.bb?h=scarthgap#n18). So, even if all packages in `packagegroup-arago-tisdk-graphics` are added unconditionally, the inclusion of entire `packagegroup-arago-tisdk-graphics` is bounded by a condition.

Reply:
Ahhh 
Well the commit msg in meta-arago threw me off then... it still says addons, while you are adding it to graphics packagegroup. 

In that case LGTM